<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStdAction</class>
 <widget class="QDialog" name="CStdAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Standard action</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_in">
     <property name="title">
      <string>In</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_out">
     <property name="title">
      <string>Out</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="m_response">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_invoke">
       <property name="text">
        <string>Invoke</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
